style.css

```diff
@@ -0,0 +1,92 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+main {
+  border-bottom-left-radius: 25px;
+}
+
+.portfolio-nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 72px;
+}
+
+.portfolio-nav--logo {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.portfolio-nav--nav-options {
+  display: flex;
+  list-style: none;
+}
+
+.portfolio-nav--nav-options li {
+  padding: 0 20px;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+}
+
+.portfolio-nav--nav-options a {
+  text-decoration: none;
+}
```

Comment:
Please make sure that the `ul` tag with class `portfolio-nav--nav-options` has a `display` property of `none` because it should not be displayed on a mobile screen:

The Figma design:

![figma](https://user-images.githubusercontent.com/95412943/212044997-0129d9b7-a282-44fa-b5e0-b6f547354922.PNG)

Your design:

![student](https://user-images.githubusercontent.com/95412943/212045046-7e6779b7-cdbf-427f-bc4f-7f2dd5807902.PNG)